db/migrate/20150508144336_add_attachment_profile_header_to_users.rb

```diff
@@ -1,7 +1,10 @@
 class AddAttachmentProfileHeaderToUsers < ActiveRecord::Migration[4.2]
   def change
     change_table :users do |t|
-      t.attachment :profile_header
+      t.string :profile_header_file_name
+      t.string :profile_header_content_type
+      t.bigint :profile_header_file_size
```

Comment:
This was a `t.integer` before -- please change it (and create a migration for that if it really should become a `bigint`)